src/SourceGenerators/Uno.UI.SourceGenerators.net6.Tests/ImplementedRoutedEventsGeneratorTests/Given_ImplementedRoutedEventsGenerator.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System.Collections.Immutable;
+using System.Text;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.CodeAnalysis.Text;
+using Uno.UI.SourceGenerators.ImplementedRoutedEvents;
+using Uno.UI.SourceGenerators.Tests.Verifiers;
+
+namespace Uno.UI.SourceGenerators.Tests.ImplementedRoutedEventsGeneratorTests
+{
+	using Verify = CSharpSourceGeneratorVerifier<ImplementedRoutedEventsGenerator>;
+
+	[TestClass]
+	public class Given_ImplementedRoutedEventsGenerator
+	{
+		private static readonly ReferenceAssemblies s_defaultWithUno = ReferenceAssemblies.Default.AddPackages(
+			ImmutableArray.Create(new PackageIdentity("Uno.UI", "4.4.20"))); // UPDATE with a version that has a UIElementGeneratedProxy available.
+
+		// TODO: Remove and use s_defaultWithUno when we have a version with UIElementGeneratedProxy available.
```

Comment:
@jeromelaban Yes. Just to get rid of the fake `UIElementGeneratedProxy` we define in the test. There is no way to address that in this PR other than trying to consume local Uno.UI packages instead of from NuGet, but I think it's not worth doing that. Following up should be okay for now.